### DISCUSSION TITLE
When I'm trying to create the instance, the security group dropdown list doesn't contain 'allow-all'.

### DISCUSSION THREAD

**Discussion by Jyothsnamandlipalli** [learndevopsonline/learndevopsonline#465] [Q&A]
![image](https://user-images.githubusercontent.com/129792325/230738375-80975871-bec8-46e7-89cb-27a3acb81cff.png)

Hope, the 'allow-all' security group has been created already for everyone use.

**Comment by noolivara** [comment]
No, Manually you need to crate allow-all security group.

**Reply by Jyothsnamandlipalli** [reply]
Thank you
